ddtrace/llmobs/_trace_processor.py

```diff
@@ -86,8 +86,13 @@ def _llmobs_span_event(self, span: Span) -> Dict[str, Any]:
             meta["output"]["value"] = span._meta.pop(OUTPUT_VALUE)
         if span_kind == "retrieval" and span.get_tag(OUTPUT_DOCUMENTS) is not None:
             meta["output"]["documents"] = json.loads(span._meta.pop(OUTPUT_DOCUMENTS))
-        if span_kind == "llm" and span.get_tag(INPUT_PROMPT) is not None:
-            meta["input"]["prompt"] = json.loads(span._meta.pop(INPUT_PROMPT))
+        if span.get_tag(INPUT_PROMPT) is not None:
+            if span_kind != "llm":
+                log.warning(
```

Comment:
Can we still pop `INPUT_PROMPT` from the span meta? We want to avoid submitting temporary tags to APM.

Reply:
Thanks for catching!